src/app/parcels/parcelsTable/ParcelsPage.tsx

```diff
@@ -301,10 +341,44 @@ const ParcelsPage: React.FC = () => {
         );
     };
 
+    const packingManagerViewDataPortion: ParcelsTableRow[] = parcelsDataPortion.filter((parcel) => {
+        if (dayjs(parcel.packingDate) === today && parcel.packingSlot !== "AM") {
+            return;
+        }
+        if (dayjs(parcel.packingDate) === yesterday && parcel.packingSlot !== "PM") {
+            return;
+        }
+        if (!parcel.allStatuses) {
+            return;
+        }
+        if (parcel.allStatuses.includes("Out for Delivery")) {
+            return;
+        }
+        if (
+            parcel.allStatuses.includes("Shipping Labels Downloaded") &&
+            parcel.allStatuses.includes("Shopping List Downloaded") &&
+            parcel.allStatuses.includes("Called and Confirmed")
+        ) {
+            return parcel;
+        }
+    });
+
     return (
         <>
             <PreTableControls>
                 <ActionsContainer>
+                    <Button
+                        variant={isPackingManagerView ? "outlined" : "contained"}
+                        onClick={() => setIsPackingManagerView(false)}
+                    >
+                        All parcels
+                    </Button>
+                    <Button
+                        variant={isPackingManagerView ? "contained" : "outlined"}
+                        onClick={() => setIsPackingManagerView(true)}
+                    >
+                        Packing manager view
+                    </Button>
```

Comment:
I think we want these button outside the `ActionsContainer`, on the far left of the `PreTableControls` bar, you'll need to make a new wrapper component and probably change the styling for `PreTableControls` and maybe `ActionsController`